packages/wp-source/src/actions.ts

```diff
@@ -29,20 +29,37 @@ const actions: WpSource["actions"]["source"] = {
     // Get options
     const force = options ? options.force : false;
 
-    if (!data || force) {
-      // Add the attributes that should be present even if fetch fails or we throw a ServerError below
+    if (!data) {
+      // If there is no data yet, just set the necessary flags:
       source.data[link] = {
-        link,
-        route: linkParams.route,
-        query,
-        page,
         isFetching: true,
         isReady: false,
       };
-    } else if (data.isReady || data.isFetching || data.isError) {
+    } else if (force) {
+      // If we fetch with `{ force: true }`, then only set `isFetching` to true again
+      data.isFetching = true;
+
+      // This is a workaround in case that `data` has previously included an error
+      if (data.isError) {
+        delete data.isError;
+        delete data.errorStatus;
+        delete data.errorStatusText;
```

Comment:
This is going to break the themes, because data is going to be ready (`data.isReady`) but with no information about what it is. You should reset the data object completely. Also, could you please add a test of that case?

Reply:
I'm sorry but I don't really understand. 

> You should reset the data object completely
My understanding was that we are not supposed to reset the data object but just remove the error as we discussed?

> This is going to break the themes, because data is going to be ready (data.isReady === true) but with no information about what it is. 
Can you elaborate?


> Also, could you please add a test of that case?
done.

Reply:
> My understanding was that we are not supposed to reset the data object but just remove the error as we discussed?

But in the case of the error, you can break themes and packages because the data is going to be this!

```js
{
  isReady: true;
  isFetching: true;
}
```

So please, in case of a previous error, reset the data completely back to `isReady: false, isFetching: true`.